src/node_modules/app/components/Nav.svelte

```diff
@@ -32,15 +33,27 @@
 		<header>
 			{segment ? capitalize(segment) : 'Home'}
 		</header>
-		<div on:click={handleClick}>
-			{#if isMenuOpen}
+		{#if !screen?.sizeFlags.medium}
+			<div
+				on:click={toggleA11yMenu}
+				class='clickable'
+			>
+				<Icon
+					glyph={A11y}
+					size='34'
```

Comment:
(1) size should be 24 like all the other icons so we can remove this line as it's `Icon`'s default